hub/core/dataset/dataset.py

```diff
@@ -506,9 +506,6 @@ def create_tensor(
         if is_link and (
             sample_compression != UNSPECIFIED or chunk_compression != UNSPECIFIED
         ):
-            warnings.warn(
-                "Chunk_compression and sample_compression aren't valid for tensors with linked data. Ignoring these arguments."
```

Comment:
we don't only have to suppress warning, this entire if block needs to go otherwise they will be set as unspecified